gapis/resolve/command_tree.go

```diff
@@ -236,8 +243,11 @@ func (r *CommandTreeResolvable) Resolve(ctx context.Context) (interface{}, error
 			return fmt.Errorf("Fail to mutate command %v: %v", cmd, err)
 		}
 		if filter(id, cmd, s, api.SubCmdIdx([]uint64{uint64(id)})) {
+			var groupnum int
```

Comment:
`groupnum` doesn't appear to be used anywhere.

Reply:
Leftover instrumentation, sorry.